mnefun/_mnefun.py

```diff
@@ -1730,12 +1731,13 @@ def do_preprocessing_combined(p, subjects, run_indices):
                   '        %s' % bad_file)
             if not op.isdir(bad_dir):
                 os.mkdir(bad_dir)
-            # do autobad
+            # do autobad without applying eeg average ref projection
```

Comment:
Read with add=False, mark bad, then add projector. When it gets added at
that stage it shouldn't include the bads